Tests/TuistLoaderTests/Loaders/CachedManifestLoaderTests.swift

```diff
@@ -399,3 +453,9 @@ final class CachedManifestLoaderTests: TuistUnitTestCase {
         }
     }
 }
+
+extension NIOFileSystem.FileSystemError: @retroactive Equatable {
```

Comment:
```suggestion
extension NIOFileSystem.FileSystemError: Equatable {
```

We can't use `@retroactive` because the release of the CLI still uses Xcode 15 where this annotation is not available afaik.